public/js/app/components/__tests__/CourseFileLinks.test.js

```diff
@@ -61,7 +61,12 @@ describe('Component <CourseFileLinks>', () => {
     const propsWithMemoWebPage = {
       courseCode: 'KIP1111',
       language,
-      courseRound: { roundId: '7', round_course_term: ['1970', '1'], has_round_published_memo: true },
+      courseRound: {
+        roundId: '7',
```

Comment:
`roundId` has to be removed to be sure tests working properly

Reply:
Removed